site/content/posts/theory-of-operation.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Cartographer: Theory of Operation"
+slug: theory-of-operation
+date: 2022-02-28
+author: Rasheed Abdul-Aziz
+authorLocation: https://github.com/squeedee
+image: /img/posts/theory-of-operation/cover-image.png
+excerpt: "Theory Of Operation"
+tags: ['faq', 'theory']
+---
+
+# Assumptions
+
+To explain Cartographer's operation, we're going to make some assumptions:
+
+1. You are familiar with creating and modifying Kubernetes resources
+2. You know the difference between a job and a pod.
```

Comment:
In this context, would the difference be Job+ short running process vs Pod= long running process?

Reply:
yes. I will get to terminating (short lived) processes later when I introduce runnable